pkg/operator/ceph/pool/controller.go

```diff
@@ -340,8 +340,12 @@ func (r *ReconcileCephBlockPool) reconcile(request reconcile.Request) (reconcile
 
 		// If not mirrored there is no Status Info field to fulfil
 	} else {
+		mirroringEnabled, err := r.isAnyRadosNamespaceMirrored(poolSpec.Name)
+		if mirroringEnabled {
+			return reconcile.Result{}, *cephBlockPool, errors.Wrap(err, "mirroring is enabled in the pool radosnmamespace, so it should be disabled in the pool spec")
```

Comment:
Can we include the name of at least one of the rados namespaces that still had it enabled?
```suggestion
			return reconcile.Result{}, *cephBlockPool, errors.Wrap(err, "mirroring still must be disabled in all radosnamespaces in the pool before disabling mirroring in the pool, found enabled for namespace %q", radosNamespaceEnabled)
```

Reply:
it wont be confusing in that case?
Either we should add all of then or just mention disable it

Reply:
IMO it's more helpful to mention one instead of none. Then if they forgot to disable it on a single rados namespace, we will help them find it. But not a big deal to keep your error message as it is.